add 2015 B/C 50ns and 25ns RelVal workflows

hengne opened the pull request:
Adding 2015 B/C 50ns and 25ns RelVal workflows, including porting back some rearrangement of the RelVal scripts that applied in 75x and 76x.

hengne committed:
run 2015b/c json file updated up to data cert HN on 27 aug. 2015

hengne committed:
port back from 75x 2015b/c data workflows from developement on 75x branch https://github.com/hengne/cmssw/tree/from-CMSSW_7_5_2_relval_newData_fullList , not fully finished, to be tested.

hengne committed:
port back missing module, continue task in commit f005b52ffbdac416598ea1af9e3420d81e0c4196

hengne committed:
change run2_hlt GT from GR_H_V58A to 74X_dataRun2_HLT_frozen_v1, for fixing a crash due to L1 menu

hengne committed:
shorten label strings to avoid workflow string length below 100 chars.

hengne committed:
reset RequestPriority back to 500000

hengne committed:
remove those additional 2015c workflows which we do not need in regular relval campaign.

hengne committed:
change autoHLT to the relval version

hengne committed:
change it to the run2_data GT for relval

hengne committed:
fix conflict, take PR 10883 as reference

cmsbuild commented:
A new Pull Request was created by @hengne (Hengne Li) for CMSSW_7_4_X.

add 2015 B/C 50ns and 25ns RelVal workflows

It involves the following packages:

Configuration/AlCa
Configuration/PyReleaseValidation
DPGAnalysis/Skims


@diguida, @boudoul, @franzoni, @cerminar, @cmsbuild, @srimanob, @mmusich can you please review it and eventually sign? Thanks.
@ghellwig, @Martin-Grunewald this is something you requested to watch as well.
You can sign-off by replying to this message having '+1' in the first line of your reply.
You can reject by replying  to this message having '-1' in the first line of your reply.
If you are a L2 or a release manager you can ask for tests by saying 'please test' in the first line of a comment.
@degano you are the release manager for this.
You can merge this pull request by typing 'merge' in the first line of your comment.

franzoni commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: d94e7430e4b815033ce94206b0f9657cee600d1d
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11113/7892/summary.html

mmusich commented on Configuration/AlCa/python/autoCond.py:
```diff
@@ -20,7 +20,7 @@
     'run1_data'         :   'GR_R_74_V12A::All',
     # GlobalTag for Run2 data reprocessing
-    'run2_data'         :   'GR_R_74_V13A::All',
+    'run2_data'         :   '74X_dataRun2_Prompt_v1::All',
```
@hengne you should change 'Configuration/AlCa/python/autoCond_condDBv2.py'. autoCond.py still maps to v1 Global Tags (and actually '74X_dataRun2_Prompt_v1::All' doesn't even exists in the condition database V1. @diguida

hengne replied:
@mmusich Hi Marco, yes, I know Configuration/AlCa/python/autoCond_condDBv2.py is actually what is used, and it is set to 
  'run2_data' : '74X_dataRun2_v1',
already as what want.

The change in autoCond.py was historically left there from my first touch of it when I still didn't know this v1/v2 difference.  Because I also know this autoCond.py is not used, so i didn't spend time to change it back to GR_R_74_V13A::All', which i guess was also not valid anymore. Do you want me to change it back to 'GR_R_74_V13A::All' ?  Let me know, i can do that right away.

mmusich replied:
yes please.

hengne replied:
done. @mmusich

On Fri, Sep 4, 2015 at 3:54 PM, Marco Musich <notifications@github.com>
wrote:

> In Configuration/AlCa/python/autoCond.py
> <https://github.com/cms-sw/cmssw/pull/11113#discussion_r38753384>:
>
> > @@ -20,7 +20,7 @@
> >      # GlobalTag for Run1 data reprocessing
> >      'run1_data'         :   'GR_R_74_V12A::All',
> >      # GlobalTag for Run2 data reprocessing
> > -    'run2_data'         :   'GR_R_74_V13A::All',
> > +    'run2_data'         :   '74X_dataRun2_Prompt_v1::All',
>
> yes please.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/11113/files#r38753384>.
>

mmusich commented:
-1
see above comment.
@diguida @ggovi I think it's time to get rid of this confusing v2 default for 7.4.X

franzoni commented:
@hengne 
let's first place a PR in 76
Once that'll be into the IB we can come back to this one

hengne commented:
yes, this work on 76x is ongoing.

On Fri, Sep 4, 2015 at 3:49 PM, Giovanni Franzoni <notifications@github.com>
wrote:

> @hengne <https://github.com/hengne>
> let's first place a PR in 76
> Once that'll be into the IB we can come back to this one
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/11113#issuecomment-137742719>.
>

hengne committed:
add lumilist selection

hengne committed:
change back to old GT for run2_data for condDB_v1

hengne committed:
merge to get two new commits.
Merge branch 'from-CMSSW_7_4_10_patch2_relval' into from-CMSSW_7_4_10_patch2_relval_rebase

hengne committed:
rebase again
Merge branch 'CMSSW_7_4_X' into from-CMSSW_7_4_10_patch2_relval_rebase

hengne commented:
supersede by https://github.com/cms-sw/cmssw/pull/11424
